android/src/org/ppsspp/ppsspp/NativeActivity.java

```diff
@@ -309,71 +286,66 @@ public void checkForVibrator() {
         }
 	}
 
-	// Override this to scale the backbuffer (use the Android hardware scaler)
-	public void getDesiredBackbufferSize(Point sz) {
-		sz.x = 0;
-		sz.y = 0;
+	private Runnable mEmulationRunner = new Runnable() {
+		@Override
+		public void run() {
+			// Bit of a hack - loop until onSurfaceCreated succeeds.
```

Comment:
Should we just defer this (set a flag) to when the surface has been created?

-[Unknown]